### ISSUE TITLE
[map] san francisco map crosswalk ids are missing

### ISSUE THREAD

**Issue by ntutangyun** [lgsvl/simulator#1916]
Hi,

I noticed that the ID field of some crosswalks are missing as shown below for the san Francisco map downloaded at [this page](https://github.com/lgsvl/apollo-5.0/tree/simulator/modules/map/data/san_francisco)

![image](https://user-images.githubusercontent.com/16061105/145705979-24ee5be8-785f-4252-92cd-eb6996a0b3c0.png)

**Comment by EricBoiseLGSVL** [comment]
Yes, some crosswalk ids are missing in SF map.  Thanks for posting
